lib/src/components/NxModal/__tests__/NxModal.test.tsx

```diff
@@ -20,56 +20,59 @@ describe('NxModal', function() {
     onClose: dummyCloseHandler
   };
 
-  const getShallow = getShallowComponent<Props>(NxModal, minimalProps),
-      getModal = (props?: Partial<Props>) => getShallow(props).children();
+  const getMounted = getMountedComponent<Props>(NxModal, minimalProps);
 
   it('renders a context provider around an nx-modal-backdrop <dialog> containing an nx-modal <div>', function () {
-    const contextProvider = getShallow(),
-        nxModal = contextProvider.children();
+    const component = getMounted(),
+        dialog = component.find('dialog');
 
-    expect(contextProvider).toMatchSelector(NxModalContext.Provider);
-    expect(nxModal).toMatchSelector('dialog.nx-modal-backdrop');
-    expect(nxModal.children()).toMatchSelector('div.nx-modal');
+    expect(dialog).toExist();
+    expect(dialog).toHaveClassName('.nx-modal-backdrop');
+    expect(dialog.children()).toMatchSelector('div.nx-modal');
   });
 
   it('renders children nodes within the modal', function() {
-    const nxModal = getModal({ children: <div className="bar"/> });
```

Comment:
Surely you could have kept a `getModal` function and the existing variable names and cut down on changes to this file?

Reply:
> Surely you could have kept a `getModal` function and the existing variable names and cut down on changes to this file?

Cleaned up here: 27c7f208a012a5a8e9466c66966156dd843e9495

Reply:
> Cleaned up here: https://github.com/sonatype/sonatype-react-shared-components/commit/27c7f208a012a5a8e9466c66966156dd843e9495

Aside from the stuff about the context, and switching from shallow to mount rendering, I'm not understanding why any of the changes that are still in that file are necessary.  We want everything about `NxModal`'s DOM to be unchanged by the PR, so its unit tests largely should be unchanged

Reply:
> Aside from the stuff about the context, and switching from shallow to mount rendering, I'm not understanding why any of the changes that are still in that file are necessary. We want everything about NxModal's DOM to be unchanged by the PR, so its unit tests largely should be unchanged

I've started over and made minimal changes in 2eb22684d

Reply:
> 2eb22684d

Sorry, 2eb22684d is now pushed.  Its diff relative to main is still more minimal than what you have.

Reply:
> > [2eb2268](https://github.com/sonatype/sonatype-react-shared-components/commit/2eb22684ddb7104a9520397e034494e3f9599e24)
> 
> Sorry, [2eb2268](https://github.com/sonatype/sonatype-react-shared-components/commit/2eb22684ddb7104a9520397e034494e3f9599e24) is now pushed. Its diff relative to main is still more minimal than what you have.

Thanks for your help. I've merged it, and fixed conflict here: 8e6d560a43f6b7319921306d784a3ea14c41454a